app/static/pages/home/LearnerInfoDefinitions.js

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+oppiaFoundationWebsite.constant('LEARNER_INFO', [
+
+  {
+    age: 12,
+    fullName: 'Student A',
+    location: 'Khanpur, Delhi',
+    profilePictureImageFilename: 'student_A_and_student_B.jpg',
```

Comment:
Can this image be cropped so that it shows on only one student? Weird to have two students in the image and only one in the quote.

Reply:
Image cropped!